lib/auth.coffee

```diff
@@ -31,6 +31,18 @@ getAuth = (deps, opts) ->
 		else
 			return err
 
+	wrapAuthFn = (eventName, fn) ->
+		return ->
+			fn.apply(authBase, arguments)
+			.finally -> pubsub.publish(eventName)
+
+	auth = Object.assign({}, authBase, {
+		setKey: wrapAuthFn('auth.keyChange', authBase.setKey)
```

Comment:
NIT: Extract the event name to a constant and use it when subscribing as well (similar to Redux actions). You can even put it in a single file that exports all events so it is also easy to keep track of what events are supported for pubsub.

Reply:
Let's keep this for when we add a second supported event, should we?